### PR TITLE
Add compatibility for symfony 5 and specify required symfony packages

### PR DESCRIPTION
**Pull request by alexander-schranz**

### CODE REVIEW

**Comment by alexander-schranz** [comment]
@dbu I'm stuck here with the following error:

> In DoctrinePHPCRExtension.php line 505:
> PHPCR ODM is activated in the config but does not seem loadable.  

Think it have todo with the `symfony-cmf/testing` update do you have any hint I'm little bit lost here 🙈

File: DependencyInjection/Configuration.php [hunk 1]
```diff
@@ -23,8 +23,15 @@ class Configuration implements ConfigurationInterface
...
     public function getConfigTreeBuilder()
     {
-        $treeBuilder = new TreeBuilder();
-        $treeBuilder->root('phpcr_migrations')
+        $treeBuilder = new TreeBuilder('phpcr_migrations');
+        if (method_exists($treeBuilder, 'getRootNode')) {
+            $root = $treeBuilder->getRootNode();
+        } else {
+            // BC layer for symfony/config 4.1 and older
+            $root = $treeBuilder->root('phpcr_migrations');
```
Comment (dantleech) [inline DependencyInjection/Configuration.php]: would it also be fine to drop <4.1 support ?

**Comment by dantleech** [comment]
Looks good to me, I'd also consider dropping < 4.1 support, but if it does no harm.

**Comment by dantleech** [comment]
Good work @alexander-schranz  :)

**Comment by alexander-schranz** [comment]
@dantleech Dropping Symfony ^4.1 support is ok for me if we have a bug we could still create a patch release for the older version. Sulu 2.0 currently does require symonfy ^4.3.

**Comment by alexander-schranz** [comment]
@dantleech let me know if I can do something.

**Pull request merged**